app/views/notifications/index.html.haml

```diff
@@ -1,8 +1,19 @@
-%article
+- content_for :title do
+  = 'Notifications'
+
+%article.user{ data: "settings"}
   %header
-    %h1 Notifications
-    %h2 What's been going on in GlitterGallery
-  %content
+    %h1
+      = avatar current_user.email
+      = 'Notifications'
```

Comment:
You don't need the `=` on this line. See the [HAML docs](http://haml.info/docs/yardoc/file.REFERENCE.html#inserting_ruby_) for more info.